zulipterminal/ui_tools/boxes.py

```diff
@@ -402,15 +402,29 @@ def main_view(self) -> List[Any]:
         soup = BeautifulSoup(self.message['content'], 'lxml')
         content = (None, self.soup2markup(soup.find(name='body')))
         active_char = '▒'  # Options are '█', '▓', '▒', '░'
+
+        message_content = dict(
+            edited_label_size=0,
+            left_padding=12
+        )
+
+        if self.message['id'] in self.model.index['edited_messages']:
+            message_content['edited_label_size'] = 11
+            message_content['left_padding'] = 1
```

Comment:
These values are fine for now, but I think we might be able to tune them further later.

Reply:
Okay :+1: